hazelcast/sql.py

```diff
@@ -1233,15 +1241,18 @@ def execute(self, sql, params, cursor_buffer_size, timeout, expected_result_type
 
         connection = None
         try:
+            try:
+                # Serialize the passed parameters.
+                serialized_params = [
+                    self._serialization_service.to_data(param) for param in statement.parameters
+                ]
+            except SchemaNotReplicatedError as e:
+                return self._send_schema_and_retry_fn(e, self.execute, sql, params, kwargs)
```

Comment:
Is this `kwargs` usage valid? 

```python3
def a(a, b, c):
     print(kwargs)

a(a = 1, b = 2, c = 3)
```
throws
```
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "<stdin>", line 2, in a
NameError: name 'kwargs' is not defined

```

To verify, can you add SQL compact tests?

Reply:
no, not at all. it was most probably a copy-paste error

Reply:
I will improve the SQL compatibility tests, but I need to update the class in the remote controller. I will do that today

Reply:
OK, I have updated the test, and found a way to use serialization on the client side without requiring a change on the remote controller classes.